src/App.scss

```diff
@@ -17,23 +28,80 @@ button {
 
 .TodoInfo {
   width: max-content;
-  padding: 8px;
-  margin: 12px 0;
-  border: 1px solid #000;
-  border-radius: 8px;
-  background-color: antiquewhite;
+  background-color: #fff7f1;
+  border: 1px solid rgba(0, 0, 0, 0.12);
+  border-radius: 4px;
+  padding: 16px;
+  margin: 1rem 0;
 
   &__title {
-    margin: 4px 0;
+    font-weight: 700;
     font-size: inherit;
+    margin-top: 0;
+    margin-bottom: 8px;
     color: #f00;
   }
 
   &--completed &__title {
+    background-color: #ebf5ee;
     color: #080;
   }
 }
 
+input::placeholder {
+  color: black;
+}
+
+.field {
+  margin-bottom: 16px;
+}
+
+label {
+  display: block;
+  margin-bottom: 4px;
+  color: rgba(0, 0, 0, 0.87);
+  font-size: 14px;
+  font-weight: 500;
+}
+
+input[type="text"],
+select {
```

Comment:
It is better to use class selectors instead of tags

Reply:
![image](https://github.com/mate-academy/react_add-todo-form/assets/114029531/ae9aec4e-9830-4237-be94-b6e04d49fc69)
tags - its default style for this task.